frontend/pages/packs/EditPackPage/components/PackQueryEditorModal/PackQueryEditorModal.tsx

```diff
@@ -140,13 +145,26 @@ const PackQueryEditorModal = ({
   };
 
   const onFormSubmit = (): void => {
+    setErrorFrequency("");
     const query_id = () => {
       if (editQuery) {
         return editQuery.query_id;
       }
       return selectedQuery?.id;
     };
 
+    const f = parseInt(selectedFrequency, 10);
+    if (!f || f < 0) {
```

Comment:
Should this be `if (!f || f < 1)`? As is, the value `0` would not trigger this error.

Reply:
Wouldn't 0 evaluate falsey so !0 would be true and trigger the error?